crates/bindings-csharp/Runtime/Internal/FFI.cs

```diff
@@ -174,13 +206,28 @@ out uint out_
     );
 
     [LibraryImport(StdbNamespace)]
-    public static partial void volatile_nonatomic_schedule_immediate(
-        [In] byte[] name,
-        uint name_len,
-        [In] byte[] args,
-        uint args_len
+    public static partial Errno bytes_source_read(
```

Comment:
Why were those moved around? It makes diff harder to read.

Reply:
Ah my bad, I made them in the same order as the wasm proposal to make it easier to see if they were all present